docs/faq.rst

```diff
@@ -391,6 +391,16 @@ How can I configure and debug the exact version exported by my package?
 
     (This requires the ``setuptools-scm`` package is installed in your environment)
 
+Why do I see the error ``AssertionError: Can't parse version ...``?
+    The way ``setuptools-scm`` parses git tags has some edge cases leading
+    to this assertion error. The best way is to either remove the tag or
+    rename it to either a proper semantic version like ``v0.4.2`` or avoid
+    using any number (e.g. ``draft``).
```

Comment:
Ronny confirmed in the PyPA discord chat that `setuptools-scm` has no intention of supporting non-version tags.

```suggestion
    rename it to a proper semantic version like ``v0.4.2``.
```